charts/gsp-cluster/templates/01-aws-system/aws-service-operator.yaml

```diff
@@ -0,0 +1,232 @@
+  metadata:
+    name: snssubscriptions.service-operator.aws
+  spec:
+    group: service-operator.aws
+    names:
+      kind: SNSSubscription
+      listKind: SNSSubscriptionList
+      plural: snssubscriptions
+      shortNames:
+      - subscription
+      singular: snssubscription
+    scope: Namespaced
+    version: v1alpha1
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: snstopics.service-operator.aws
+  spec:
+    group: service-operator.aws
+    names:
+      kind: SNSTopic
+      listKind: SNSTopicList
+      plural: snstopics
+      shortNames:
+      - sns
+      - topic
+      - topics
+      singular: snstopic
+    scope: Namespaced
+    version: v1alpha1
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: sqsqueues.service-operator.aws
+  spec:
+    group: service-operator.aws
+    names:
+      kind: SQSQueue
+      listKind: SQSQueueList
+      plural: sqsqueues
+      shortNames:
+      - sqs
+      - queue
+      - queues
+      singular: sqsqueue
+    scope: Namespaced
+    version: v1alpha1
+
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: aws-service-operator
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - secrets
+    - pods
+    - configmaps
+    - services
+    - events
+    verbs:
+    - get
+    - list
+    - watch
+    - create
+    - delete
+    - update
+    - patch
+  - apiGroups:
+    - apiextensions.k8s.io
+    resources:
+    - customresourcedefinitions
+    verbs:
+    - get
+    - list
+    - watch
+    - create
+    - delete
+  - apiGroups:
+    - service-operator.aws
+    resources:
+    - "*"
+    verbs:
+    - "*"
+
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: aws-service-operator
+    namespace: aws-service-operator
+
+- kind: ClusterRoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: aws-service-operator
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: aws-service-operator
+  subjects:
+  - kind: ServiceAccount
+    name: aws-service-operator
+    namespace: aws-service-operator
+
+- kind: Deployment
+  apiVersion: apps/v1beta1
+  metadata:
+    name: aws-service-operator
+    namespace: aws-service-operator
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        annotations:
+          iam.amazonaws.com/role: aws-service-operator
+        labels:
+          app: aws-service-operator
+      spec:
+        serviceAccountName: aws-service-operator
+        containers:
+        - name: aws-service-operator
+          image: awsserviceoperator/aws-service-operator:v0.0.1-alpha4
+          imagePullPolicy: Always
+          args:
+            - server
+            - --cluster-name=gsp
+            - --region={{ .Values.global.serviceOperatorRegion }}
+            - --account-id={{ .Values.global.serviceOperatorAccountId }}
```

Comment:
Looks like (from the master copy, haven't dug into the image itself) it can get the region from metadata, though we may need to set it explicitly blank rather than let it default (which appears to be us-west-2?).
Doesn't appear to be able to get the account ID from metadata.

Reply:
https://github.com/awslabs/aws-service-operator/pull/200